inspire/ext/formatter_jinja_filters/record.py

```diff
@@ -19,6 +19,7 @@
 
 from inspire.ext.formatter_jinja_filters.general import apply_template_on_array
 from inspire.utils.bibtex import Bibtex
+from inspire.utils.latex import Latex
```

Comment:
Ahh cool, I thought that because I do not use them anymore, I deleted them
and forgot to delete the imports. I will amend the commit.

2015-08-26 9:39 GMT+02:00 Javier Martin Montull <notifications@github.com>:

> In inspire/ext/formatter_jinja_filters/record.py
> <https://github.com/inspirehep/inspire-next/pull/320#discussion_r37953640>
> :
>
> > @@ -19,6 +19,7 @@
> >
> >  from inspire.ext.formatter_jinja_filters.general import apply_template_on_array
> >  from inspire.utils.bibtex import Bibtex
> > +from inspire.utils.latex import Latex
>
> hmm, what happened to this file? You import Latex but never use it. And
> you removed the Bibtex filter.
>
> Even if for the moment you are only using the formats via AJAX, leave both
> as a jinja2 filters as well in case we need them.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/inspirehep/inspire-next/pull/320/files#r37953640>.
>